### PR TITLE
feat: Add additional click modes to element

### PR DESCRIPTION
**Pull request by jimevanssfdc**

### CODE REVIEW

File: utam-core/src/main/java/utam/core/element/Clickable.java [hunk 1]
```diff
@@ -20,4 +22,23 @@ public interface Clickable extends BasicElement {
...
+   * class.
+   */
+  void doubleClick();
+
+  /**
+   * click and hold the element for the duration specified, using
+   * the Selenium Actions class.
+   * @param duration duration for holding the click on the element
+   */
+  void clickAndHold(Duration duration);
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/element/Clickable.java:37]: to align with JS https://github.com/salesforce-experience-platform/utam-js/pull/409/files let's use integer as parameter (duration in seconds)

File: utam-core/src/main/java/utam/core/element/Element.java [hunk 2]
```diff
@@ -74,6 +75,23 @@ public interface Element {
...
+   * perform a secondary-button click on the element
+   */
+  void rightClick();
+
+  /**
+   * click and hold the element
+   * @param duration the duration for which to hold.
+   *                 Using Duration.ZERO will hold the click indefinitely
+   */
+  void clickAndHold(Duration duration);
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/element/Element.java:88]: to align with JS https://github.com/salesforce-experience-platform/utam-js/pull/409/files let's use integer as parameter (duration in seconds)

**Review by lizaiv77 — CHANGES_REQUESTED** [review]

File: utam-core/src/main/java/utam/core/element/Element.java [hunk 3]
```diff
@@ -7,6 +7,7 @@
  */
 package utam.core.element;
 
+import java.time.Duration;
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/element/Element.java:10]: redundant import i think

File: utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java [hunk 4]
```diff
@@ -11,6 +11,7 @@
 import static utam.core.selenium.element.DriverAdapter.getNotFoundErr;
 import static utam.core.selenium.element.DriverAdapter.getSeleniumDriver;
 
+import java.time.Duration;
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java:14]: redundant import

**Review by lizaiv77 — CHANGES_REQUESTED** [review]
minor comments abt redundant import

File: utam-core/src/main/java/utam/core/element/Element.java (hunk 3 above)
Comment (jimevanssfdc) [reply utam-core/src/main/java/utam/core/element/Element.java:10]: This one I missed when I removed them. Fixed in next commit.

File: utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java (hunk 4 above)
Comment (jimevanssfdc) [reply utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java:14]: This one is required; `clickAndHold` uses it to change the integer number of seconds into a `Duration` of seconds for how long to hold the click.

**Review by lizaiv77 — APPROVED** [review]
LGTM

**Pull request merged**